### ISSUE TITLE
How to do enum with bridge 

### ISSUE THREAD

**Issue by Noobware1** [ethanblake4/dart_eval#120]
As the title says. I want register a enum in a bridge.

**Comment by ethanblake4** [comment]
See [here](https://github.com/ethanblake4/flutter_eval/blob/f20ab67d04dff6d528cad9158b508a3c25f5202c/lib/src/sky_engine/ui/text.dart#L35) for a sample enum definition.

Then you register it on the compiler with `defineBridgeEnum` and on the runtime with `registerBridgeEnumValues` [like this](https://github.com/ethanblake4/flutter_eval/blob/f20ab67d04dff6d528cad9158b508a3c25f5202c/lib/flutter_eval.dart#L308).

**Comment by ethanblake4** [comment]
Yes, the main example for dart_eval on pub.dev has an example of an abstract `$Bridge` class and you can [see flutter_eval](https://github.com/ethanblake4/flutter_eval/blob/f20ab67d04dff6d528cad9158b508a3c25f5202c/lib/src/rendering/object.dart#L9C63-L9C67) for examples of an abstract wrapper. For the most part, you just have to put `BridgeClassDef(BridgeClassType($type, isAbstract: true)`
